Makefile

```diff
@@ -40,8 +40,8 @@ lint: $(VENV)/pyvenv.cfg
 .PHONY: reformat
 reformat: $(VENV)/pyvenv.cfg
 	. $(VENV_BIN)/activate && \
-		ruff format src/ && \
-		ruff check --fix src/
+		ruff format --target-version py39 src/ && \
+		ruff check --fix --target-version py39 src/
```

Comment:
I believe these `--target-version` flags can be removed, since `ruff` should figure out the version from the `pyproject.toml` itself.

Reply:
You're right https://docs.astral.sh/ruff/settings/#target-version